gemrb/core/Store.cpp

```diff
@@ -125,27 +125,25 @@ int Store::AcceptableItemType(ieDword type, ieDword invflags, bool pc) const
 		ret &= ~IE_STORE_BUY;
 
 	if (pc && Type < StoreType::BG2CONT) {
-		//don't allow selling of non destructible items
-		if (!(invflags&IE_INV_ITEM_DESTRUCTIBLE )) {
-			ret &= ~IE_STORE_SELL;
-		}
 
 		//don't allow selling of critical items (they could still be put in bags) ... unless the shop is special
-		if ((invflags&IE_INV_ITEM_CRITICAL) && !(Flags&IE_STORE_BUYCRITS)) {
+		if ((invflags&IE_INV_ITEM_CRITICAL) && (!(Flags&IE_STORE_BUYCRITS))) {
```

Comment:
This doesn't change anything. Also I'm not sure why you're adding so many parentheses everywhere. Two pairs are enough in this line.